examples/plotting/customjs_expr.py

```diff
@@ -1,3 +1,10 @@
+''' This example demonstrates using CustomJS Expression to create a sinusoidal line chart with Bokeh.
```

Comment:
"with Bokeh" is redundant :) 
```suggestion
''' This example demonstrates using CustomJS Expression to create a sinusoidal line chart.
```